python/ray/train/tests/test_checkpoint_manager.py

```diff
@@ -181,6 +181,72 @@ def test_nested_get_checkpoint_score(metrics):
     assert manager._get_checkpoint_score(tracked_checkpoint) == (True, 5.0)
 
 
+@pytest.mark.parametrize("has_score_attr", [True, False])
+def test_only_store_score_attr(has_score_attr, checkpoint_paths, monkeypatch):
+    monkeypatch.setenv("RAY_TRAIN_ONLY_STORE_CHECKPOINT_SCORE_ATTRIBUTE", "1")
+
+    # Set up CheckpointManager with 1 checkpoint.
```

Comment:
What does 1 checkpoint mean here?

Reply:
I meant that I wanted to
* Insert 1 checkpoint first
* Verify that we can insert 1 checkpoint before and after it.
But on second thought I think that's unnecessarily confusing and changed the comments.